pkg/runtime/runtime.go

```diff
@@ -258,6 +258,9 @@ type pubsubSubscribedMessage struct {
 func newDaprRuntime(runtimeConfig *internalConfig, globalConfig *config.Configuration, accessControlList *config.AccessControlList, resiliencyProvider resiliency.Provider) *DaprRuntime {
 	ctx, cancel := context.WithCancel(context.Background())
 
+	wfe := wfengine.NewWorkflowEngine(wfengine.NewWorkflowConfig(runtimeConfig.id))
+	wfe.ConfigureGrpcExecutor()
```

Comment:
It looks like in all cases we call `ConfigureGrpcExecutor()` immediately after creating the engine. Any reason not to just put this logic inside `NewWorkflowEngine(...)' to avoid the extra explicit step?

Reply:
Ah, or is it because you need to configure an alternate executor in the test code? In that case, I think this separation makes sense.

Reply:
Correct, I originally put it inside NewWorkfowEngine but then realized I needed a different one for unit tests